packages/main/test/pages/TabContainer.html

```diff
@@ -26,14 +26,63 @@
 	<header class="header">
 		<h1 class="header-title">ui5-tabcontainer</h1>
 	</header>
+	<section>
+		<ui5-toggle-button id="densityToggle">Compact</ui5-toggle-button>
```

Comment:
The density toggle button should already work across the whole page, why is it neccessary to add it again here? Also, it's an element with a duplicate ID.